src/run/formatters.ts

```diff
@@ -33,9 +35,8 @@ export async function initializeFormatters({
     target: string,
     type: string
   ): Promise<Formatter> {
-    stream.on('error', (error) => {
-      console.error(error.message)
-      process.exit(1)
+    stream.on('error', (error: Error) => {
+      stderr.write(error.message)
```

Comment:
No longer exiting on stream error. Don't think thats a big deal as we are still writing to stderr. We could inject a exit function if we still want to exit here.